src/rust/engine/process_execution/src/remote.rs

```diff
@@ -354,7 +354,16 @@ impl CommandRunner {
         })
         .to_boxed()
     } else {
-      future::ok(Bytes::from(execute_response.get_result().get_stdout_raw())).to_boxed()
+      let stdout_raw = Bytes::from(execute_response.get_result().get_stdout_raw());
+      let stdout_copy = stdout_raw.clone();
+      self
+        .store
+        .store_file_bytes(stdout_raw, true)
+        .map_err(move |error| {
+          ExecutionError::Fatal(format!("Error storing raw stdout: {:?}", error))
+        })
+        .map(|_| stdout_copy)
```

Comment:
@illicitonion : Based on what you were saying about the necessity of `move` for closures, this shouldn't work... but it seems to be. I think that this implies that unambiguous moves are executed automatically?